x-pack/plugin/sql/qa/mixed-node/src/test/java/org/elasticsearch/xpack/sql/qa/mixed_node/SqlSearchIT.java

```diff
@@ -256,20 +251,33 @@ private void assertAllTypesWithNodes(Map<String, Object> expectedResponse, List<
         ) {
             @SuppressWarnings("unchecked")
             List<Map<String, Object>> columns = (List<Map<String, Object>>) expectedResponse.get("columns");
-            String intervalYearMonth = "INTERVAL '150' YEAR AS interval_year, ";
-            String intervalDayTime = "INTERVAL '163' MINUTE AS interval_minute, ";
```

Comment:
I find this style to be more "visible" and easier to read. Can you, please, keep the original version?

Reply:
sure, I inlined it because I thought it was a leftover from using a conditional here in earlier revisions.